itests/testdata/reissue_tx_testdata.go

```diff
@@ -0,0 +1,337 @@
+			true,
+			ReissueExpectedValuesNegative{
+				WavesDiffBalance:  0,
+				AssetDiffBalance:  0,
+				Reissuable:        true,
+				ErrGoMsg:          errMsg,
+				ErrScalaMsg:       errMsg,
+				ErrBrdCstGoMsg:    errBrdCstMsg,
+				ErrBrdCstScalaMsg: "failed to parse json message",
+			}),
+		"Invalid fee (0 < fee < min)": *NewReissueTestData(
+			utl.GetAccount(suite, 2),
+			assetID,
+			10,
+			utl.GetCurrentTimestampInMs(),
+			TestChainID,
+			10000000,
+			true,
+			ReissueExpectedValuesNegative{
+				WavesDiffBalance:  0,
+				AssetDiffBalance:  0,
+				Reissuable:        true,
+				ErrGoMsg:          errMsg,
+				ErrScalaMsg:       errMsg,
+				ErrBrdCstGoMsg:    errBrdCstMsg,
+				ErrBrdCstScalaMsg: "(10 in WAVES) does not exceed minimal value of 100000 WAVES",
+			}),
+		"Invalid fee (fee = 0)": *NewReissueTestData(
+			utl.GetAccount(suite, 2),
+			assetID,
+			0,
+			utl.GetCurrentTimestampInMs(),
+			TestChainID,
+			10000000,
+			true,
+			ReissueExpectedValuesNegative{
+				WavesDiffBalance:  0,
+				AssetDiffBalance:  0,
+				Reissuable:        true,
+				ErrGoMsg:          errMsg,
+				ErrScalaMsg:       errMsg,
+				ErrBrdCstGoMsg:    errBrdCstMsg,
+				ErrBrdCstScalaMsg: "insufficient fee",
+			}),
+		"Reissue token when there are not enough funds on the account balance": *NewReissueTestData(
+			utl.GetAccount(suite, 2),
+			assetID,
+			uint64(100000000+utl.GetAvailableBalanceInWavesGo(suite, utl.GetAccount(suite, 2).Address)),
+			utl.GetCurrentTimestampInMs(),
+			TestChainID,
+			10000000,
+			true,
+			ReissueExpectedValuesNegative{
+				WavesDiffBalance:  0,
+				AssetDiffBalance:  0,
+				Reissuable:        true,
+				ErrGoMsg:          errMsg,
+				ErrScalaMsg:       errMsg,
+				ErrBrdCstGoMsg:    errBrdCstMsg,
+				ErrBrdCstScalaMsg: "Accounts balance error",
+			}),
+		"Timestamp more than 7200000ms in the past relative to previous block timestamp": *NewReissueTestData(
+			utl.GetAccount(suite, 2),
+			assetID,
+			100000,
+			utl.GetCurrentTimestampInMs()-7215000,
+			TestChainID,
+			10000000,
+			true,
+			ReissueExpectedValuesNegative{
+				WavesDiffBalance:  0,
+				AssetDiffBalance:  0,
+				Reissuable:        true,
+				ErrGoMsg:          errMsg,
+				ErrScalaMsg:       errMsg,
+				ErrBrdCstGoMsg:    "",
+				ErrBrdCstScalaMsg: "is more than 7200000ms in the past relative to previous block timestamp",
+			}),
+		"Timestamp more than 5400000ms in the future relative to previous block timestamp": *NewReissueTestData(
+			utl.GetAccount(suite, 2),
+			assetID,
+			100000,
+			utl.GetCurrentTimestampInMs()+54160000,
+			TestChainID,
+			10000000,
+			true,
+			ReissueExpectedValuesNegative{
+				WavesDiffBalance:  0,
+				AssetDiffBalance:  0,
+				Reissuable:        true,
+				ErrGoMsg:          errMsg,
+				ErrScalaMsg:       errMsg,
+				ErrBrdCstGoMsg:    "",
+				ErrBrdCstScalaMsg: "is more than 5400000ms in the future relative to block timestamp",
+			}),
+		/*"Custom chainID": *NewReissueTestData(
```

Comment:
Do we need this test data if it's commented out?

Reply:
Add new negative data matrix for this cases and add small fix in test